service/project.go

```diff
@@ -370,6 +370,48 @@ func (s *ProjectService) RejectInvitation(ctx context.Context, tkn cryptox.Token
 	return s.repo.DeleteInvitation(ctx, invitation.ID)
 }
 
+func (s *ProjectService) ListMembers(ctx context.Context, projectID, authUserID uuid.UUID) ([]model.ProjectMember, error) {
+	if err := s.authGuard.AuthorizeAdminRole(ctx, authUserID); err != nil {
+		return nil, err
+	}
+
+	exists, err := s.projectExists(ctx, projectID)
+	if err != nil {
+		return nil, err
+	}
+	if !exists {
+		return nil, apierrors.NewProjectNotFoundError()
+	}
+
+	return s.repo.ReadMembersForProject(ctx, projectID)
+}
+
+func (s *ProjectService) DeleteMember(ctx context.Context, projectID, userID, authUserID uuid.UUID) error {
+	if err := s.authGuard.AuthorizeAdminRole(ctx, authUserID); err != nil {
+		return err
+	}
+
+	exists, err := s.projectExists(ctx, projectID)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		return apierrors.NewProjectNotFoundError()
+	}
```

Comment:
Actually, a similar thing here.
Meanwhile, the project might get deleted, soo this check is not sharp enough.

In the case of member deletion, I think everything can be simplified to one call and returning a `ProjectMemberNotFoundError`.

But when it comes to listing.. see next comment...